app/resources/api/v1/development_resource.rb

```diff
@@ -17,6 +17,8 @@ class DevelopmentResource < JSONAPI::Resource
       boolean_filters :rdv, :asofright, :ovr55, :clusteros, :phased,
         :stalled, :cancelled, :hidden
 
+      filter :status
```

Comment:
There are still some remaining filters to be added - namely `name` (🙃), municipality, and project status. I'm also not quite sure what the handling is for whether a search is `saved: true` or not. Let's talk...